packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -65,6 +69,37 @@ export default class PhoneNumberInput extends React.Component<Props, State> {
     }
   }
 
+  async triggerPhoneNumberRequest() {
+    try {
+      const phone = await SmsRetriever.requestPhoneNumber()
+      const phoneNumber = parsePhoneNumber(phone, '')
+
+      if (phoneNumber) {
+        this.setState({
+          phoneNumber: phoneNumber.displayNumber.toString(),
+        })
+        if (phoneNumber.countryCode) {
+          // TODO known issue, the country code is not enough to
```

Comment:
How should we handle this?
Should we pick the first one and use that? Should we not autofill country when there's ambiguity?
Also what does `getCountryByPhoneCountryCode` return when there's ambiguity? an array?

Reply:
It's typed as `LocalizedCountry`, so we only get one, the result is arbitrary based on `getCountryByPhoneCountryCode ` implementation.  The `getCountryByPhoneCountryCode` function is conceptually wrong because it claims doing something that's not possible returning just one value. Apparently USA and Canada are the only two countries that share this problem.

Reply:
I researched a little bit more and it looks like to do the differentiation is possible but 
`libphonenumber` does not support it in js: 

`PhoneNumberOfflineGeocoder - provides geographical information related to a phone number.` https://github.com/ruimarinho/google-libphonenumber/blob/master/README.md#installation

Reply:
Ok. if it's only US and Canada then it's probably okay to leave the TODO and circle back to this later. I can look at it when I'm in Canada ;)